src/NRediSearch/Aggregation/AggregationBuilder.cs

```diff
@@ -0,0 +1,157 @@
+﻿// .NET port of https://github.com/RedisLabs/JRediSearch/
```

Comment:
wait; is this class the same thing as `AggregationRequest` ? they *look* pretty similar..?

Reply:
I thought the same exact thing. 

While I was reviewing the Java implementation I noticed that the `aggregate` method that accepted an instance of `AggregationRequest` had been deprecated in favor of the the `AggregationBuilder`.

Reply:
@gkorland? Any insight from the Java side of things?

Reply:
Yes AggregationRequest was replaced by AggregationBuilder